config/config.go

```diff
@@ -54,3 +54,8 @@ var ImportArweaveGatewayURLs []*url.URL
 var PrivateBucketURL *url.URL
 
 var HTTPInternalAddress string
+
+var DefaultSourcePlaybackHosts = map[string]string{
+	"lp-us-catalyst-recordings-com.storage.googleapis.com":     "//recordings-cdn.lp-playback.studio/hls",
```

Comment:
Even though it's not strictly secret, I don't love having these exposed. Could we just fallback to some default behaviour if the CLI param isn't present?

Reply:
I think we'd have to just disable source playback in that case otherwise it'd require some tricky logic to then ensure the source copy does get done. I think that would be ok as we'd still have a big speed improvement from dropping the copy, but in theory we should be able to roll the cli param out at the same time and this won't be a problem.